schema.sql

```diff
@@ -8,3 +8,48 @@
     neutered boolean,
     weight_kg integer
 );
+
+ALTER TABLE animals
+ADD COLUMN species_id INT,
+ADD COLUMN owner_id INT;
+
+CREATE TABLE owners (
+    id  INT generated by default as identity primary key, 
+    full_name VARCHAR(100), 
+    age INT
+);
+
+CREATE TABLE species (
+    id  INT generated by default as identity primary key,
+    name VARCHAR(100)
+);
+
+
+CREATE TABLE owners (
+    id  INT PRIMARY KEY,
+    full_name VARCHAR(100),
+    age INT
+ );
+
+CREATE TABLE species (
+     id SERIAL PRIMARY KEY,
+     name VARCHAR(100)
+ );
+
+ALTER TABLE animals
+DROP COLUMN species;
+
+ALTER TABLE animals
+ADD CONSTRAINT fk_species_id
+FOREIGN KEY (species_id) REFERENCES species(id);
+
+ALTER TABLE animals
+ADD CONSTRAINT fk_owner_id
+FOREIGN KEY (owner_id) REFERENCES owners(id);
+
+INSERT INTO species (name)
+VALUES ('Pokemon');
+
+INSERT INTO species (name)
+VALUES ('Digimon');
```

Comment:
- [ ] Like the previous reviewer mentioned, anything to do with data insertion should be written in the `data.sql` file. You should remove this block of code here.

Reply:
- Removed please

![image](https://github.com/AbuTalha3/vet-clinic-database/assets/127955219/8f95782c-2af9-493a-a1d5-e7dded96a4f1)